packages/fxa-shared/nestjs/sentry/reporting.ts

```diff
@@ -71,9 +83,14 @@ export function filterSentryEvent(event: Sentry.Event, hint: unknown) {
       event.request.url = event.request.url.replace(TOKENREGEX, FILTERED);
     }
     if (event.request.query_string) {
-      event.request.query_string = (
-        event.request.query_string as string
-      ).replace(TOKENREGEX, URIENCODEDFILTERED);
+      if (typeof event.request.query_string === 'string') {
+        event.request.query_string = event.request.query_string.replace(
+          TOKENREGEX,
+          URIENCODEDFILTERED
```

Comment:
Raw query string gets a URI encoded filter, filterObject doesn't need it as its processed further when its not the raw string.